modules/CmpdRegBulkLoader/src/server/routes/CmpdRegBulkLoaderRoutes.coffee

```diff
@@ -166,28 +196,38 @@ exports.registerCmpds = (req, resp) ->
 			else
 				fileName = req.body.fileName
 				delete req.body.fileName
-				config = require '../conf/compiled/conf.js'
-				baseurl = config.all.client.service.cmpdReg.persistence.fullpath+"bulkload/registerSdf"
-				request = require 'request'
-				request(
-					method: 'POST'
-					url: baseurl
-					body: req.body
-					json: true
-				, (error, response, json) =>
-					console.log json
-					if !error && response.statusCode == 200 && json.reportFiles?
-						createSummaryZip fileName, json
-					else
-						console.log 'got ajax error trying to register compounds'
-						console.log error
-						console.log json
-						console.log response
-						if json.summary?
-							resp.json [json]
+				exports.getScientistsInternal (authors) =>
+					_ = require 'underscore'
+
+					authorCodes = _.pluck authors, "code"
+
+					exports.validationPropertiesInternal req.body, (sdfProperties) =>
```

Comment:
It's great to add this upfront validation of the Scientists referenced in the file. What do you think about doing the same thing for Stereo Categories and Projects?

Reply:
Will resolve with #554